srcpkgs/bitwig-studio/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'bitwig-studio'
+pkgname=bitwig-studio
+version=3.2.6
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+depends="xdg-utils xcb-util-wm libbsd"
+short_desc="DAW for music production, remixing and live performance"
+maintainer="Alessandro Saglia <ale.saglia@gmail.com>"
+license="custom:bitwig"
+homepage="bitwig.com"
```

Comment:
This should be an URL, i.e. `https://bitwig.com/`

Reply:
Fixed with last commit